hadoop-hdds/interface-client/src/main/proto/proto.lock

```diff
@@ -1476,6 +1476,21 @@
           }
         ],
         "messages": [
+          {
```

Comment:
The proto change seems unrelated to this PR.

Reply:
It was auto generated. I did not modify anything here in this file.

Reply:
Integration test failure seems unrelated to the changes here.

Reply:
> It was auto generated. I did not modify anything here in this file.

Please exclude both proto.lock files from the patch.

Reply:
> > It was auto generated. I did not modify anything here in this file.
> 
> Please exclude both proto.lock files from the patch.

done

Reply:
> The proto.lock change seems unrelated to this PR.

these files are reverted.